### DISCUSSION TITLE
Leaving the project

### DISCUSSION THREAD

**Discussion by peepo5** [FreeTubeApp/FreeTube#2166] [General]
Due to various personal reasons, I will be leaving this project and I would like to request @PrestonN to remove my permissions.

**Comment by PrestonN** [comment]
I've gone ahead and removed you from the project. I'd like to thank you for the work that you've done and wish you the best of luck. Let me know if you ever have time again and I'll be happy to add you again.

**Comment by anonaddict** [comment]
Thank you for your service
